browser.js

```diff
@@ -69,6 +69,8 @@ function reload (config) {
   background.onload = () => {
     var renderer = createRenderer(opts);
 
+    document.querySelector('.download').onclick = () => window.open(renderer.canvas.toDataURL(), '_blank')
```

Comment:
`canvas` is already defined in line 9.
You can use it directly without changing `createRenderer.js`.